enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -214,7 +214,7 @@ class Meta:
                 'trim_whitespace': True,
             },
             'description': {
-                'allow_blank': False,
+                'allow_blank': True,
```

Comment:
[curious] Should the `description` also now include `null=True` to store `NULL` in the database when left blank?

Reply:
Django docs discourages allowing both blank (empty string) and null values for any char/text fields to avoid confusion on how to represent (and filter for) "no content".  They specifically recommend always using `null=False`, and `blank` can be anything depending on what's needed.